Examples/StenographerExample/Tests iOS/Tests_iOSLaunchTests.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Tests_iOSLaunchTests.swift
+//  Tests iOS
+//
+//  Created by Ryan Goodlett on 1/12/22.
+//
+
+import XCTest
+
+class Tests_iOSLaunchTests: XCTestCase {
+
+    override class var runsForEachTargetApplicationUIConfiguration: Bool {
+        true
+    }
+
+    override func setUpWithError() throws {
+        continueAfterFailure = false
+    }
+
+    func testLaunch() throws {
+        let app = XCUIApplication()
+        app.launch()
+
+        // Insert steps here to perform after app launch but before taking a screenshot,
+        // such as logging into a test account or navigating somewhere in the app
+
+        let attachment = XCTAttachment(screenshot: app.screenshot())
+        attachment.name = "Launch Screen"
+        attachment.lifetime = .keepAlways
+        add(attachment)
+    }
```

Comment:
Same for this. Until there are meaningful UI tests, this will just slow things down.